### DISCUSSION TITLE
Welcome to h5bench Discussions!

### DISCUSSION THREAD

**Discussion by sbyna** [hpc-io/h5bench#111] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lizhiyuell** [comment]
Hi, I'm Zhiyue Li, and currently I'm working on HPC I/O optimizations. H5bench is really a convenient tool to test various parallel I/O kernels. However, one thing confuses me is that some of the benchmarks do not report the overall performance (e.g. I/O bandwidth), like the AMReX, MACSio and Exerciser workloads. Passing the "csv-file" cannot generate the relevant .csv file. How I can get a metric for the overall performance? Thanks!
